docs/markdown/Docker/docker.md

```diff
@@ -61,31 +61,39 @@ docker_image(
 > 🚧 The `docker_image` `instructions` field
 > 
 > Each `docker_image` uses a `Dockerfile` referred to by the `source` field, unless you have provided a value to the `instructions` field.
-> 
-> When using the `instructions` field, make sure that the default value for `source` does not match a file, or there will be a conflict about which information to use.
 
 Adding dependencies to your `docker_image` targets
 --------------------------------------------------
 
-A Dockerfile executes in a _context_ - a set of files that the commands in the Dockerfile can reference, e.g., by copying them into the image).
+A Dockerfile is built in a _context_ - a set of files that the commands in the Dockerfile can reference, e.g., by copying them into the image.
 
-When you run `docker` directly, the context is usually a directory within your repo. That directory must contain the Dockerfile (typically at the root of the context) and any files that the build requires. If those files are themselves the product of a build step, or if they are sources from elsewhere in the repo, then you have to copy them into the context.
+When you run `docker build` directly, the context is usually a directory within your repo containing the Dockerfile (typically at the root of the context) and any files that the build requires. If those files were themselves the product of a build step, or if they were sources from elsewhere in the repo, then you would have to copy them into the context.
```

Comment:
When I read this paragraph the first time, I read it as something that would need to be configured _with_ pants rahter than without - trying to clear that up a bit.

Reply:
hmm... I agree, and I still feel that "then you would have to copy them into the context." could be a bit misleading. Not really sure what to do with it though..